test/lint/lint-spelling.ignore-words.txt

```diff
@@ -1,17 +1,16 @@
 hights
 mor
-mut
-objext
 useable
 wit
 unparseable
-copyable
 cachable
-errorstring
-keyserver
-homogenous
-setban
 hist
 ser
-unselect
-lowercased
+asend
+blockin
+fo
+fpr
+inout
+keypair
+nin
+unser
```

Comment:
nit: The sorting mode should be lexicographical or "natural" (i.e., as reported by the linter, that is lexicographical for containing file paths), but not a mix of both.

Reply:
Thanks, I decided for the lexicographical order as I think it's easier to read and maintain. Done.